ports/stm32/boards/common_isr_ram/common_isr.ld

```diff
@@ -0,0 +1,40 @@
+/* This linker script fragment is intended to be included in SECTIONS. */
+
+/* The startup code goes first into FLASH */
+.isr_vector :
+{
+    . = ALIGN(4);
+    __isr_vector_ram_start = .;
+    KEEP(*(.isr_vector)) /* Startup code */
+
+    /* These functions need to run from ram to enable uart
+       reception during flash erase/write operations.
+       Defining them here ensures they're copied from
+       flash (in ram_irq.c) along with the isr_vector above.
+    */
+    . = ALIGN(4);
+    *(.text.pendsv_kbd_intr)
+    *(.text.pendsv_schedule_dispatch)
+    *(.text.storage_systick_callback)
+    *(.text.SysTick_Handler)
+    *(.text.uart_irq_handler)
+    *(.text.UART*_IRQHandler)
+    *(.text.USART*_IRQHandler)
+    *(.text.FLASH_PageErase)
+    *(.text.FLASH_SectorErase)
+    *(.text.FLASH_WaitForLastOperation)
+    *(.text.HAL_FLASHEx_Erase)
+    *(.text.HAL_GetTick)
+
+    __isr_vector_ram_end = .;
+    . = ALIGN(4);
+
+} >RAM AT >FLASH_ISR
+
+_start_init_isr = LOADADDR(.isr_vector); /* Used by the start-up code to initialise data */
```

Comment:
looks like this variable is unused

Reply:
Ah you're right, seems I had a few different attempts worth of variables defined in here still sorry!